main.nf

```diff
@@ -913,7 +944,7 @@ process multiqc {
     file ('picard/*') from picard_reports.collect()
     file ('deeptools/*') from deepTools_multiqc.collect()
     file ('phantompeakqualtools/*') from spp_out_mqc.collect()
-    file ('phantompeakqualtools/*') from calculateNSCRSC_results.collect()
+    file ('phantompeakqualtools/*') from spp_csv_mqc.collect()
```

Comment:
I guess you intend to add to the MultiQC module for this data?

Because it's generated by a custom R script, not taken directly from the tool, it shouldn't be in the main MultiQC module. Instead, add headers to the csv file and take it in as custom content.

Alternatively, put the functionality that's in the R script into the MultiQC module, then everyone can use it :)